.travis.yml

```diff
@@ -113,6 +113,66 @@ matrix:
         - ICC1=gcc-6
```

Comment:
3.7.1 and 3.8.1 tests should be removed.

Reply:
We won't be removing these yet because LLVM >= 4 is still in the "experimental" stage.

I added a note to #2371 to remove these when that is resolved.